debug/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/launchConfigurations/LaunchConfigurationTabGroupViewer.java

```diff
@@ -1606,6 +1605,29 @@ protected void errorDialog(CoreException exception) {
 		ErrorDialog.openError(getShell(), null, null, exception.getStatus());
 	}
 
+	/**
+	 * @param tabKind The class of the tab that has to be activated.
+	 * @return <code>true</code> if a tab was activated, <code>false</code>
+	 *         otherwise.
+	 */
```

Comment:
Please improve the documentation. It should be clear that the first of the tabs with a matching kind is activated. In case there is no tab of that kind, no activation is performed, and in case there are multiple the first is taken.

Reply:
See https://github.com/eclipse-platform/eclipse.platform/pull/823